lib/brakeman/report/report_text.rb

```diff
@@ -19,7 +19,7 @@ def generate_report
     add_chunk generate_controllers if tracker.options[:debug] or tracker.options[:report_routes]
     add_chunk generate_templates if tracker.options[:debug]
     add_chunk generate_obsolete
-    add_chunk generate_errors 
```

Comment:
Whoops, I forgot that my editor trims trailing whitespace. Let me know if you want me to remove these edits from the commit!

Reply:
Not a problem, I generally clean up as I go, too.